cypress/support/step_definitions/testSetup.md

```diff
@@ -1,15 +1,36 @@
 # TestSetup Glue
-## I initialize a {string} test
 
-### Purpose: initalize a test during prerequisite
+## the environment has been set up for {string} tests
+
+### Purpose: Sets up the environment for the specified test based on the test name. 
+
+#### Below are the tasks that are performed based on the test name:
+- This glue will used to initialize the firebolt object for the dynamic calls.
+  - Passing the key name in `module:method` format
+    ##### Format
+    ```javascript
+    Given the environment has been set up for 'module:method' tests
+    ```
+    - In the abover format, `module` is the name of the Firebolt object and firebolt object fetched and stored in the `runtime` environment variable using module name.
+    - `module:method` this will split the key name by `:` and both the module and method will be stored in the `runtime` environment variable.
+  - Passing the key name directly
+    ##### Format
+    ```javascript
+    Given the environment has been set up for 'keyName' tests
+    ```
```

Comment:
what about space in between? Accessibility Closedcaptionsettings

Reply:
With space also will work